### DISCUSSION TITLE
HMIP RFUSB Stick in Raspberrymatic über VM in Proxmox Keine Gerätekopplung

### DISCUSSION THREAD

**Discussion by brunsheinz** [OpenCCU/OpenCCU#2568] [🇩🇪 Fragen&Antworten]
Hallo Ich bin gerade dabei meine Homematic ccu3 mit Raspberrymatic auf einen Proxmox Server umzustellen . Nach der Installation habe ich das Backup meiner CCU3 eingespielt . Ich möchte den RFUSB Stick nutzen . Der RFUSB Stick wird auch erkannt als Gerät ,aber er findet keine Geräte . Erst hatte ich bei Anwahl von CuxD Probleme , habe auf CuxD Version X86 umgestellt ( Intel Celeron im PC) . Dort steht oben :eQ-3 HmIP-RFUSB [FF] - no driver .Ich habe noch eine CCU2 als Lan Gateway und einen HMIP HAP als Gateway in der Anlage . Lan Gateway CCU2 erkennt alle im Bereich liegenden Geräte . Habe auch schon versucht ein HMIP Gerät nach Reset neu anzulernen , hat auch nicht funktioniert . Habe ich vielleicht beim Löten des Sticks einen Fehler gemacht ? Die Lötpunkte habe ich auch schon mal überprüft ,konnte aber nichts feststellen . Wer kann mir da weiter helfen . Die Logdatei  hänge ich mal hier an .
[ccu3-webui-2023-12-16.log](https://github.com/jens-maus/RaspberryMatic/files/13693540/ccu3-webui-2023-12-16.log)

**Comment by Baxxy13** [comment]
Das Hauptproblem beim HmIP-RFUSB sind tatsächlich schlechte Lötungen. 
Ich hatte da mal Informationen im [Homematic-Forum](https://homematic-forum.de/forum/viewtopic.php?f=31&t=79828) zusammengetragen. Darin ist auch ein Link zu einer Anleitung mit der man relativ gut die Funktionalität des Sticks checken kann.

Möglicherweise hast du aber andere Probleme.
Dein `hmserver.log` zeigt ungewöhnliche Fehlermeldungen die ich erstmal nicht deuten kann.
Außerdem sieht man in der `boot.log` das dein `rfd` nicht startet.
`Starting rfd: ....................ERROR`

**Comment by brunsheinz** [comment]
Hallo Ich habe heute noch mal alle Lötpunkte auf dem Stick nachgelötet . Der Fehler :Starting rfd lag bei mir , da ich bei der Installation nicht die Tab Taste gedrückt hatte und somit der Stick wohl nicht installiert wurde . Habe noch eine neue VM in Proxmox erstellt ohne ein Backup einzuspielen . Dort habe ich versucht ein HMIP-SWDO und ein HM-CCI-3_FM anzulernen , ohne Erfolg . Ich habe davon nochmal eine Log Datei gespeichert . Habe versucht nach der Info zu suchen wie in den Anleitungen beschrieben wird , aber ich kann da nichts erkennen , bin aber auch kein IT Experte
[homematic-raspi-2023-12-18 (1).log](https://github.com/jens-maus/RaspberryMatic/files/13702755/homematic-raspi-2023-12-18.1.log)

**Reply by brunsheinz** [reply]
Hallo Nach nochmaliger Kontrolle der Platine habe ich eine kleine Lötzinn Ablagerung am SMD Chip auf der Platine gefunden und beseitigt . Also der Stick funktioniert , alles O.K. Entschuldigung nochmal für mein vorschnelles Handeln . 
![SSLO5430](https://github.com/jens-maus/RaspberryMatic/assets/95573353/f7c9dae6-5b7f-41e6-a3eb-ea6b3fad3b20)

**Reply by Baxxy13** [reply]
Sehr gut. Dann kannst du ja jetzt "losfunken". 😉 
Im Log war übrigens keinerlei Kommunikation zu erkennen, aber das hattest du ja selbst schon herausgefunden.
